profiles/common/modules/features/multisite_wysiwyg/ckeditor/plugins/link_hreflang/plugin.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * @file
+ * CKEditor Link hreflang code.
+ */
+
+(function($) {
+  CKEDITOR.plugins.add("multisite_wysiwyg_link_hreflang", {
+    init: function(editor) {
+      CKEDITOR.on("dialogDefinition", function(e) {
+        if ((e.editor != editor) || (e.data.name != 'link')) {
+          return;
+        }
+
+        // Overrides definition.
+        var definition = e.data.definition;
+
+        // Removing the "lang" input and adding the "hreflang" one
+        // in the advanced tab.
+        var advancedTab = definition.getContents("advanced");
+        // Removing "lang" input.
+        advancedTab.remove("advLangCode");
+
+        // Adding "hreflang" input.
+        advancedTab.add({
+          type: "text",
+          id: "multisite_wysiwyg_link_hreflang",
+          label: Drupal.t("Language code (hreflang)"),
+          title: Drupal.t("Language code (hreflang): the code is on 2 characters (ex: en)."),
```

Comment:
I agree on the text but not the dropdown because it is out of the ticket scope. The ticket only asks to replace the "lang" attribute by the "hreflang" one. Adding a validation or a dropdown is going far beyond what it is requested.